app/components/ch/components/Btn.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <button type="button" :class="classes">
+  <button type="button" :class="classes" aria-label="btn">
```

Comment:
same here, otherwise impaired people will hear «btn» in place of «button» with their screen readers

Reply:
And on this component, an aria-label is only useful when we display the icon only. Please check if the button is «icon-only» and put text in the aria-label only in this case.